app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.ParkingSystemJava">
-        <activity android:name=".MainActivity">
+        <activity android:name="com.onboarding.estacionAR.activities.MainActivity">
```

Comment:
here you can add only `.activities.MainActivity`

Reply:
Done.